lib/activerecord-clean-db-structure/tasks/clean_db_structure.rake

```diff
@@ -9,7 +9,8 @@ Rake::Task['db:structure:dump'].enhance do
     databases = ActiveRecord::Tasks::DatabaseTasks.setup_initial_database_yaml
     ActiveRecord::Tasks::DatabaseTasks.for_each(databases) do |spec_name|
       Rails.application.config.paths['db'].each do |path|
-        filenames << File.join(path, spec_name + '_structure.sql')
+        filename = spec_name == 'primary' ? 'structure.sql' : spec_name + '_structure.sql'
```

Comment:
@RKushnir My apologies for missing this PR until now.

I'm wondering if there is a better way to determine whether this is the default name, or someone calling their database "primary" (or is that treated special by Rails and not usable as a database name?)

Reply:
Hi Lukas. I no longer remember my thinking at the moment of writing that, but I did a quick search now and it seems that Rails indeed treats "primary" as a special name. Here's what they say in the [guide for using multiple databases with activerecord](https://guides.rubyonrails.org/active_record_multiple_databases.html):
> If a primary configuration is provided, it will be used as the "default" configuration. If there is no configuration named "primary", Rails will use the first configuration as default for each environment. The default configurations will use the default Rails filenames. For example, primary configurations will use schema.rb for the schema file, whereas all the other entries will use [CONFIGURATION_NAMESPACE]_schema.rb for the filename.

Now this makes me wonder about the case when there is no "primary" database. Probably the code needs to be adjusted to use the unprefixed filename for the first database in the list.

Reply:
cc @andyatkinson since I saw you maintain this now :)